stdlib/asm/crypto/fri/frie2f4.masm

```diff
@@ -262,82 +248,52 @@ end
 #!
 #! The memory referenced above is used contiguously, as follows:
 #!
-#!   [layer_ptr ... rem_ptr ... query_start_ptr ... query_end_ptr]
+#!   [query_ptr ... layer_ptr ... rem_ptr ...]
 #!
 #! This means for example that:
 #! 1. rem_ptr - 1 points to the last (alpha0, alpha1, t_depth, d_size) tuple.
-#! 2. The length of the remainder codeword is 2 * (rem_ptr - query_start_ptr).
+#! 2. layer_ptr - 1 points to the last (e0, e1, p, poe) tuple.
 #!
 #! Cycles: for domains of size 2^n where:
-#! - n is even: 12 + 6 + num_queries * (40 + num_layers * 76 + 69) + 2626
-#! - n is odd:  12 + 6 + num_queries * (40 + num_layers * 76 + 69) + 1356
+#! - n is even: 6 + 7 + num_queries * (40 + num_layers * 76 + 26)
+#! - n is odd:  6 + 7 + num_queries * (40 + num_layers * 76 + 26)
```

Comment:
Since this doesn't depend on even/odd any more, we can probably re-write this. Also, I think `6 + 7` should actually be `7 + 4` (i.e., 7 cycles before the loop and 4 cycles after the loop).

Also, somewhere in the description of this procedure, we should probably mention that this does not verify degree of the remainder.

Reply:
Done!